ceph-maintainer-tool/config/ceph-maintainer-tool.yml

```diff
@@ -0,0 +1,21 @@
+- job:
+    name: ceph-maintainer-tool
+    project-type: freestyle
+    defaults: global
+    display-name: 'ceph-maintainer-tool'
+    block-downstream: false
+    block-upstream: false
+    scm:
+    - git:
+        url: git@github.com:ceph/ceph.git
```

Comment:
So this job will only be scanning ceph?
Seems like it makes this job less flexible. Is that what you want?

Reply:
My approach was to clone the maintainer tool repo and to get the script and run it against the ceph repo.
To test it on my local machine I tried to get this approach.
If there is any other suggestion could you please let me know.